.claude/skills/auto-claude-rdr/skill.md

```
@@ -0,0 +1,663 @@
+```
+
+**Settings:**
+
+- `reopenCommand`: User's OS-specific command to start Auto-Claude
+- `buildCommand`: Build command (default: `npm run build`)
+
+**Note:** VSCode/Claude Code sets `ELECTRON_RUN_AS_NODE=1` which breaks Electron apps. The Windows PowerShell command above clears this env var before starting.
+
+## Decision Tree
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)